commit 5fdecc66adc73381707248d275ca76577c867d6d
Author: Martin Gräßlin <mgraesslin@kde.org>
Date:   Mon Jan 13 19:54:26 2014 +0100

    Use KDE4WORKSPACE_VERSION as base for kwin's version number
    
    KWin used KDE_VERSION_STRING as version number. This means if compiled
    against kdelibs 4.12 the version is 4.12 although the true version is
    4.11.something. By using the KWIN_VERSION_STRING we can base it on the
    version number set in kde-workspace's CMakeLists.txt.
    
    REVIEW: 115002

diff --git a/kwin/main.cpp b/kwin/main.cpp
index 1052295..f75b919 100644
--- a/kwin/main.cpp
+++ b/kwin/main.cpp
@@ -456,7 +456,7 @@ void Application::resetCrashesCount()
 
 } // namespace
 
-static const char version[] = KDE_VERSION_STRING;
+static const char version[] = KWIN_VERSION_STRING;
 static const char description[] = I18N_NOOP("KDE window manager");
 
 extern "C"
